src/test/scala/com/cognite/sdk/scala/v1/DataPointsTest.scala

```diff
@@ -17,4 +20,76 @@ class DataPointsTest extends SdkTest with DataPointsResourceBehaviors[Long] {
   }
 
   it should behave like dataPointsResource(client.dataPoints)
+
+  it should "be possible to correctly query aggregate values" in {
+    val aggregatePointMap = client.dataPoints.queryAggregatesById(
+      1580330145648L,
+      0L,
+      1564272000000L,
+      "1d",
+      Seq("average", "stepInterpolation")
+    )
+
+    aggregatePointMap.get("average").value.map(x => x.timestamp).tail should contain theSameElementsInOrderAs
+        aggregatePointMap.get("stepInterpolation").value.map(x => x.timestamp)
+
+    aggregatePointMap.get("sum") should be(None)
+    aggregatePointMap.get("average").value.last.value should equal(73.9999423351708)
+
+    val aggregatePointExternalIdMap = client.dataPoints.queryAggregatesByExternalId(
+      "test__constant_74_with_noise",
+      0L,
+      1564272000000L,
+      "1d",
+      Seq("average", "stepInterpolation")
+    )
+
+    aggregatePointExternalIdMap.get("average").value.map(x => x.timestamp).tail should contain theSameElementsInOrderAs
```

Comment:
I put .tail here because average contains one extra datapoint in this case than stepInterpolation. If I take the .tail away the test fails because of the extra data point